plugins/obs-filters/cmake/nvidia.cmake

```diff
@@ -0,0 +1,6 @@
+if(ENABLE_NVAFX)
+  target_sources(obs-filters PRIVATE noise-suppress-filter.c)
+  target_enable_feature(obs-filters LIBNVAFX_ENABLED HAS_NOISEREDUCTION)
+else()
+  target_disable_feature(obs-filters "NVIDIA Audio FX support")
```

Comment:
If you choose to directly call `target_compile_definitions`, you don't need this `else()` branch.

Reply:
right